### PR TITLE
allow setting the service account email when creating a google batch CE

### LINKED ISSUE #316: Add the headJobServiceAccountEmail to Google Batch Advanced Options
Add a new option to specify an email as the Head Job Service Account Email property in Google Batch Advanced Options

### PR DESCRIPTION
**Pull request by JaimeSeqLabs**
## Description

Closes #316 

Allow setting the service account email address as configuration field when creating a new Google Batch compute environment.

## Guidelines for testing

Create a new Google Batch CE setting the service account email address:

```
$> ./tw compute-envs add google-batch -n GoogleBatchTest -c JaimeGoogleCredsDev --work-dir=gs://jaime-test-workdir --location=europe-west2 --service-account-email=<YOUR_SERVICE_ACCOUNT_EMAIL>
```

### CODE REVIEW

**Review by swampie — APPROVED** [review]

**Review by swampie — APPROVED** [review]

**Review by tcrespog — APPROVED** [review]
Solved conflicts. Tested and works.

File: .sdkmanrc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Enable auto-env through the sdkman_auto_env config
 # Add key=value pairs of SDKs to use below
-java=17.0.8-graalce
+java=21.0.4-graal
```
Comment (pgeadas) [inline .sdkmanrc:3]: This is not a CE version of the VM.

Comment (tcrespog) [reply .sdkmanrc:3]: It seems Java 21 is being used as of [recently](https://github.com/seqeralabs/tower-cli/blob/949e6ebbd3cc502d45306ad4f572695ecf1f98ac/build.gradle#L118-L121). In GitHub Actions, we are using GraalVM 21 "non-CE" [too](https://github.com/seqeralabs/tower-cli/blob/949e6ebbd3cc502d45306ad4f572695ecf1f98ac/.github/workflows/build.yml#L26-L30). Here, I've just updated `.sdkmanrc` to align them accordingly. In fact, with GraalVM 21 CE it doesn't seem to work.

image

I'll start a discussion about this.

Comment (pditommaso) [reply .sdkmanrc:3]: We can use it under the new Graalvm license 

> Subject to the conditions in the license, including the License for Early Adopter Versions, the GFTC is intended to permit use by any user including commercial and production use. Redistribution is permitted as long as it is not for a fee

https://www.graalvm.org/faq/

**Pull request merged**